docker-compose.decoupled.yaml

```diff
@@ -236,89 +288,68 @@ services:
             - ./backend-services/configuration/configs/:/etc/artemis/
             - ./backend-services/fileobserver/entrypoint:/root/entrypoint
             - ./backend-services/fileobserver/core/observer.py:/root/core/observer.py
-    notifier:
-        image: artemis_notifier
-        build: ./backend-services/notifier
-        container_name: notifier
+    mitigation:
+        image: inspiregroup/artemis-mitigation:${SYSTEM_VERSION}
+        container_name: mitigation
         restart: always
         depends_on:
             - rabbitmq
+            - database
+            - prefixtree
         expose:
             - 3000
         ports:
-            - "3010:3000"
+            - "3011:3000"
         networks:
             - artemis
         environment:
-            CONFIGURATION_HOST: configuration
-            HIJACK_LOG_FILTER: ${HIJACK_LOG_FILTER}
-            MODULE_NAME: notifier
+            DATABASE_HOST: database
+            MODULE_NAME: mitigation
```

Comment:
wouldn't this make sense to be hard-coded in the module or in the container?

Reply:
You are right, also the names of the other services. Will add this in TODO to do in the end.

Reply:
TODO: [] Hard-code module/service names in-module (applies also to the rest yaml files)

Reply:
Done in latest commit (plus renamed to `SERVICE_NAME` which is the correct term). Resolving.